checkbox-provider-dss/units/test-plan.pxu

```diff
@@ -5,6 +5,8 @@ include:
   intel_gpu_plugin/microk8s.*
   intel_gpu_plugin/daemonset_check.*
   intel_gpu_plugin/dss_.*
+  ipex.*
+  itex.*
```

Comment:
Do you have any plans or thoughts about consolidating the naming of our tests? They feel inconsistent at the moment. Also some of the other naming like the script `gpu-plugin-validation` we should also update? Doesn't have to be in this PR, but just thought I'd mention it while it's top of mind.

Reply:
Let's discuss and address this in a later PR